demos/common/models/src/openpose_decoder.cpp

```diff
@@ -1,16 +1,27 @@
-// Copyright (C) 2018-2019 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
+/*
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
We are in 2021 now. Other files require the same update.

Reply:
Updated.